frontend/talentsearch/src/js/components/diversityEquityInclusion/dialogs/VisibleMinorityDialog.tsx

```diff
@@ -0,0 +1,114 @@
+import React from "react";
+import { useIntl } from "react-intl";
+import { FormProvider, useForm, type SubmitHandler } from "react-hook-form";
+
+import Dialog from "@common/components/Dialog";
+import { Checkbox } from "@common/components/form";
+
+import type { EquityDialogProps, EquityDialogFooterProps } from "../types";
+
+import DialogActions from "./DialogActions";
+
+// Question: Is this the correct URL?
+const statsCanLink = (...chunks: string[]) => (
+  <a href="https://www23.statcan.gc.ca/imdb/p3Var.pl?Function=DEC&Id=45152">
```

Comment:
@Jerryescandon could you please confirm the correct link?
Also, should this be a regular link or an "open in a new tab" link?

Reply:
> Also, should this be a regular link or an "open in a new tab" link?

Just noting that if this is the expected behaviour we will also need to add a `(opens in new tab)` to the link text to meet WCAG 2.0.

_"For those not using assistive technology, the indication would also be available from the link text."_ - [WCAG 2.0 Techniques H83](https://www.w3.org/TR/WCAG20-TECHS/H83.html)

Reply:
Imma be real with y'all. In the next couple of days Imma have to make a new issue to update the copy on these modals. Which includes the 'href' which we're gonna link to. SO. this one seems correct for this version. BUT ALSO. I would definitely prefer it opens in a new tab.

Reply:
Thanks Jerbo.  I'm going to consider "having the correct href" and "opening in a new tab" out of scope for now then.